.github/workflows/get_issue_from_pr.yml

```diff
@@ -13,34 +13,40 @@ on:
       issue_from_pr:
         description: "The issue that the PR relates to"
         value: ${{ jobs.get_issue_from_pr.outputs.job_issue_id }}
-
+      author:
+        description: "The author of the PR"
+        value: ${{ jobs.get_issue_from_pr.outputs.author }}
 jobs:
   get_issue_from_pr:
     name: Get issue from PR
     runs-on: ubuntu-latest
     # Map the job outputs to step outputs
     outputs:
       job_issue_id: ${{ steps.get_issue.outputs.step_issue_id }}
+      author: ${{ steps.get_issue.outputs.author }}
     steps:
       - id: get_issue
         env:
           GITHUB_TOKEN: ${{ secrets.GHPROJECT_TOKEN }}
           ORGANIZATION: ${{ github.repository_owner }}
           REPO_ORG_NAME: ${{ github.repository }} # Contains the repo_name in format org/repo_name
+          PULL_REQUEST_NUMBER: ${{ github.event.pull_request.number }}
         run: |
           gh api graphql -f query='
-            query($org: String!, $repo: String!){
+            query($org: String!, $repo: String!, $number: Int!){
               repository(owner: $org, name: $repo) {
-                pullRequest(number: ${{ inputs.pr_id }}) {
+                pullRequest(number: $number) {
```

Comment:
Should: With this change `inputs.pr_id` is never used. We should either remove it and so this is hardcoded to always use the current PR or we should keep using it and pass it in on `add_assignee_when_pr_opened.yml`